biogrid/libs/biogrid-simulator/src/lib/config/efficiency.ts

```diff
@@ -0,0 +1,63 @@
+import { RESISTANCE } from ".";
+import { Power } from '@biogrid/grid-simulator';
+/**
+ * This function calculates the resistance of a given length of WIRE 16 AWG which is the one we are using for our transmission
+ * @param length the length of the wire which is synonymous to the distance between the supplier and reciever of power
+ * @returns the resistance of the wire for calculate the power loss
+ */
+export function calculateResistance(length: number, isMetre: boolean = true) {
+  if (!isMetre) {
+    length = length / 1000;
+  }
+  return RESISTANCE.RESISTANCE_16 * length;
+}
+
+/**
+ * This function calculates the current passing through the circuit at any given time
+ * @param voltage is the voltage of the circuit at that particular point
+ * @param resistance is the resistance of the wires and any connected materials in the circuit
+ * @returns the current flowing in that particular circuit
+ */
+export function calculateCurrent(voltage: number, loadResistance: number, wireResistance: number) {
+  return voltage / (loadResistance + wireResistance);
+}
+
+/**
+ * This function calculates the voltage loss while passing through the wires
+ * @param current is the current in the circuit
+ * @param resistance is the resistance of the wires
+ * @returns the voltage lost while transport through the wires
```

Comment:
The function comment and @returns comment are essentially the same thing. Maybe we can delete one of them?

Reply:
I will see if I can change the description of the function and add more details